tests/lib/store.sh

```diff
@@ -37,14 +37,34 @@ init_fake_refreshes(){
 }
 
 make_snap_installable(){
+    ACK=true
+    while [ $# -gt 0 ]; do
+        case "$1" in
+            (--noack) ACK=false; shift;;
```

Comment:
huh honestly I never knew you could have a leading `(` like that, I always assumed you had to write it like

```bash
    while [ $# -gt 0 ]; do
        case "$1" in
            --noack) ACK=false; shift;;
```
TIL :smiley: